proxy-robots-module/src/main/java/orwell/proxy/robot/RobotFactory.java

```diff
@@ -0,0 +1,33 @@
+package orwell.proxy.robot;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import orwell.proxy.config.ConfigCamera;
+import orwell.proxy.config.ConfigTank;
+import orwell.proxy.config.IConfigCamera;
+
+import java.net.URL;
+
+/**
+ * Created by Michaël Ludmann on 5/21/15.
+ */
+public final class RobotFactory {
+    private final static Logger logback = LoggerFactory.getLogger(RobotFactory.class);
+
+    public static LegoTank getLegoTank(final ConfigTank configTank) {
```

Comment:
Since you have an interface for the robots I am not sure you should have a method named after a child class.
Maybe you could use the abstract factory pattern.